docs/operator-manual/config-management-plugins.md

```diff
@@ -327,6 +327,8 @@ If you don't need to set any environment variables, you can set an empty plugin
     and let the automatic discovery to identify the plugin.
 !!! note
     If a CMP renders blank manfiests, and `prune` is set to `true`, Argo CD will automatically remove resources. CMP plugin authors should ensure errors are part of the exit code. Commonly something like `kustomize build . | cat` won't pass errors because of the pipe. Consider setting `set -o pipefail` so anything piped will pass errors on failure.
+!!! note
+    Although this should never happen, if a CMP command fails to gracefully exit on `ARGOCD_EXEC_TIMEOUT`, it will be forcefully killed after an additional timeout of `ARGOCD_EXEC_FATAL_TIMEOUT`. This is an implementation detail that should generally not concern end users.
```

Comment:
```suggestion
    If a CMP command fails to gracefully exit on `ARGOCD_EXEC_TIMEOUT`, it will be forcefully killed after an additional timeout of `ARGOCD_EXEC_FATAL_TIMEOUT`. 
```

I don't think we need so much commentary there. Here people are making CMPs so they're buying their own problems.

Reply:
thanks no worries pushed on the last commit https://github.com/argoproj/argo-cd/pull/22713/commits/a615cf49fc8485c4b8077b2c5d76dcb0fabdb0b7

Reply:
updated PR body with a new link to the docs @todaywasawesome